pylint/extensions/check_mccabe.py

```diff
@@ -0,0 +1,210 @@
+# -*- coding: utf-8 -*-
+"""Module to add McCabe checker class for pylint. """
+
+import ast
+
+from mccabe import McCabeChecker, PathGraph as PathGraphAstroid, PathGraphingAstVisitor
+from pylint.checkers.base import BaseChecker
+from pylint.checkers.utils import check_messages
+from pylint.interfaces import HIGH, IAstroidChecker
+
+
+class PathGraph(PathGraphAstroid):
+    def __init__(self, node):
+        super(PathGraph, self).__init__(name='', entity='', lineno=1)
+        self.root = node
+
+
+class McCabeASTVisitor(PathGraphingAstVisitor):
+    def __init__(self):
+        super(McCabeASTVisitor, self).__init__()
+        self.node = None
```

Comment:
Are these satisfying any API requirement from PathGraphingAstVisitor? If not, can they be made private?